changelogs/fragments/362-helm-has_plugin-fix.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - helm - make helm-diff plugin detection more reliable by splitting by any whitespace instead of explicit whitespace (`\s`)
```

Comment:
Just a couple minor nits. Since this gets turned into rst those should be double backticks. Could you also add the link to this PR at the end in parentheses? There are a few examples here: https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment.

Thanks for your work on this!